tests/functional/lang/eval-fail-recursion.err.exp

```diff
@@ -1,9 +1,9 @@
 error:
        … in the right operand of the update (//) operator
-         at /pwd/lang/eval-fail-recursion.nix:2:11:
+         at /pwd/lang/eval-fail-recursion.nix:2:14:
```

Comment:
Now taking the position from the operand itself, instead of taking it from the `//` operator (where it coincides with the lhs).
The referent of "at" was already ambiguous, so no extra change needed in this message.

Reply:
cool!